integration/run/run_test.go

```diff
@@ -850,7 +852,7 @@ func TestDeployParam(t *testing.T) {
 	assert.Equal(t, "5", appInstance.Status.AppSpec.Containers["foo"].Environment[0].Value)
 }
 
-func TestDefaultClusterComputeClass(t *testing.T) {
+func TestUsingComputeClasses(t *testing.T) {
```

Comment:
Refactored this test to run the test cases as table entries and do the same cases for both cluster and project ComputeClasses.